api_tests/src/actors/defaults.ts

```diff
@@ -37,7 +37,7 @@ export function defaultLedgerDescriptionForLedger(ledger: LedgerKind): Ledger {
         case LedgerKind.Ethereum: {
             return {
                 name: LedgerKind.Ethereum,
-                chain_id: 17,
+                chain_id: 1337,
```

Comment:
Are you able to replace with a dynamic value, similar to `nodeConfig.chain_id` in `config.ts`?

Reply:
Mhmm.. this particular one is the _default_ setting. I could make it a global constant but that be ugly here.